.azure-devops/templates/run-tests-parallel.yml

```diff
@@ -93,6 +85,26 @@ steps:
           inlineScript: |
             export COVERAGE_FILE=.coverage.${{ parameters.name }}
             pytest -vv ${{ parameters.path }} -k "_int.py" --dist=loadfile -n ${{ parameters.num_threads }} --reruns ${{ parameters.num_reruns }} --reruns-delay ${{ parameters.reruns_delay }} --cov=azext_iot --cov-config .coveragerc --junitxml=junit/test-iotext-int.xml --deselect=azext_iot/tests/iothub/core/test_iothub_discovery_int.py::TestIoTHubDiscovery::test_iothub_targets --durations=0
+        ${{ if contains(parameters.path, 'azext_iot/tests/dps') }}:
+          inlineScript: |
+            IOTHUB=`echo test-dps-hub-py$RANDOM`
+            CREATE_HUB=`grep -c "azext_iot_testhub=$" pytest.ini`
```

Comment:
I think we should not re-use the  _azext_iot_testhub_ env variable and rather add a new one for the DPS hub, to prevent iothub tests from interfering with the dps tests